pulser/simulation/simulation.py

```diff
@@ -280,7 +290,7 @@ def build_coeffs_ops(basis, addr):
         for addr in self.samples:
             for basis in self.samples[addr]:
                 if self.samples[addr][basis]:
-                    qobj_list += build_coeffs_ops(basis, addr)
+                    qobj_list += cast(list, build_coeffs_ops(basis, addr))
```

Comment:
Do you need this `cast`? The return type of `build_coeffs_ops` is already a `list`, what does this say if you remove the cast?

Reply:
Getting `error: Argument 1 to "__iadd__" of "list" has incompatible type "List[List[Any]]"; expected "Iterable[float]"`.

Reply:
Alright, leave it be then